azurerm/internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -1874,6 +1890,15 @@ func flattenKubernetesClusterManagedClusterIdentity(input *containerservice.Mana
 		identity["tenant_id"] = *input.TenantID
 	}
 
+	identity["user_assigned_identity_id"] = ""
+	if input.UserAssignedIdentities != nil {
+		keys := []string{}
+		for key := range input.UserAssignedIdentities {
+			keys = append(keys, key)
```

Comment:
Same comment as for the data source https://github.com/terraform-providers/terraform-provider-azurerm/pull/8737/files#r556381349

Reply:
https://github.com/terraform-providers/terraform-provider-azurerm/pull/8737#discussion_r556655647